app/code/Magento/Customer/Block/Account/AuthenticationPopup.php

```diff
@@ -105,14 +105,23 @@ public function getBaseUrl()
 
     /**
      * Get customer register url
-     *
+     * @deprecated
      * @return string
      */
     public function getCustomerRegisterUrlUrl()
     {
         return $this->getUrl('customer/account/create');
```

Comment:
I would call `$this->getCustomerRegisterUrl();` here, to  avoid code duplication.

Reply:
@hostep it's not applicable due to backward compatibility

Reply:
@Vasudev-22: can you take a look at this remark above also? Thanks! 🙂 

As for the deprecation notice, I would put the following, to indicate about the replacement method:
```
     * @deprecated
     * @see getCustomerRegisterUrl
```

Reply:
@hostep , Updated.
Thanks

Reply:
@hostep Yes confused those comments, Now updated. 
Thanks for pointing.